core/src/content/content.ts

```diff
@@ -101,7 +101,7 @@ export function splitOrderedName(name: string): Ordering {
   if (name.startsWith("_")) {
     return { type: "ignore" };
   }
-  if (name.endsWith(AILLY_EXTENSION)) {
+  if (name.endsWith(EXTENSION)) {
     const id = name.replace(/\.ailly$/, "");
```

Comment:
Replaces `.ailly` but the extension is variable.

Reply:
Done.